server/controllers/user.js

```diff
@@ -0,0 +1,55 @@
+import userRepo from '../repository/userRepository';
+import generateToken from '../utilities/jwtGenerator';
+import passwordUtil from '../utilities/passwordHasher';
+
+
+const { hash } = passwordUtil;
+/**
+ * User Controller class
+ */
+class User {
+  /**
+   * Register new User
+   * @param {object} request Request Object
+   * @param {object} response Response Object
+   * @returns {object} User Object
+   */
+  static async signup(request, response) {
```

Comment:
please integrate this with the validation middleware Victor wrote

Reply:
@seyi-adeleke the changes are currently on the develop branch. Once this is merged in, we would integrate the validation.

Reply:
@PriscillaSam pull develop into your branch

Reply:
@seyi-adeleke noted.